### PR TITLE
Rename entity value to entity instance

### PR DESCRIPTION
**Pull request by larisaVasile**

### CODE REVIEW

File: core/libs/editor/src/lib/editor-dialog/components/entity-instance/entity-instance-table/entity-instance-table.component.ts
Comment (michelu89) [inline core/libs/editor/src/lib/editor-dialog/components/entity-instance/entity-instance-table/entity-instance-table.component.ts:1]: Remove unused "LanguageTranslationService" import

File: core/libs/editor/src/lib/editor-dialog/components/entity-instance/utils/EntityInstanceUtil.ts
Comment (michelu89) [inline core/libs/editor/src/lib/editor-dialog/components/entity-instance/utils/EntityInstanceUtil.ts:1]: Remove unused import "property"

File: core/libs/editor/src/lib/editor-dialog/components/entity-instance/entity-instance-table/entity-instance-table.component.ts
Comment (larisaVasile) [reply core/libs/editor/src/lib/editor-dialog/components/entity-instance/entity-instance-table/entity-instance-table.component.ts:1]: done

File: core/libs/editor/src/lib/editor-dialog/components/entity-instance/utils/EntityInstanceUtil.ts
Comment (larisaVasile) [reply core/libs/editor/src/lib/editor-dialog/components/entity-instance/utils/EntityInstanceUtil.ts:1]: done

**Review by michelu89 — APPROVED** [review]

**Pull request merged**